modules/virt-accessing-vm-yaml-ssh.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vm-yaml-ssh_{context}"]
+= Accessing a virtual machine via SSH with YAML configurations
+
+You can enable a SSH connection to a virtual machine (VM) without the need to run the `virtctl expose` command. When the YAML file for the VM and the YAML file for the service are configured and applied, the service forwards the SSH traffic to the VM.
+
+The following examples show the configurations for the VM's YAML file and the service YAML file.
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+. In the YAML file for the VM, add the label and a value for exposing the service for SSH connections. Enable the `masquerade` feature for the interface.
+
++
+.Example output
+[source,yaml]
+----
+...
+kind: VirtualMachine
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: kubevirt-hyperconverged-group
+  namespace: openshift-cnv
+  labels:
+    special: vm-ssh <1>
+  name: vm-ssh
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - masquerade: {} <2>
+        name: testmasquerade <3>
+      rng: {}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  networks:
+  - name: testmasquerade
+    pod: {}
+...
+----
+<1> This label is used by the service to identify the virtual machine instances that are enabled for SSH traffic connections. The label can be any `key:value` pair that is added as a `label` to this YAML file and as a `selector` in the service YAML file.
+<2> The interface type is `masquerade`.
+<3> The name of this interface is `testmasquerade`.
+
+. Create the VM:
++
+[source,terminal]
+----
+oc create -f <path_for_the_VM_YAML_file>
```

Comment:
And because the resource created is a VirtualMachine, and not a running VirtualMachineInstance, it must be started by running:
`virtctl start vm-ssh`
Add this command right after the `oc create...` command, pleaes.

Reply:
Updated the content.